--- ganeti-p2v-transfer-0.1/instance-p2v-target/Makefile.am~	2011-08-04 09:23:48.000000000 +0200
+++ ganeti-p2v-transfer-0.1/instance-p2v-target/Makefile.am	2013-11-01 13:23:26.474636488 +0100
@@ -1,7 +1,7 @@
 osname=p2v-target
 
-osdir=$(DESTDIR)$(OS_DIR)/$(osname)
-configdir=$(DESTDIR)$(sysconfdir)/ganeti/instance-$(osname)
+osdir=$(OS_DIR)/$(osname)
+configdir=$(sysconfdir)/ganeti/instance-$(osname)
 variantsdir=$(configdir)/variants
 hookdir=$(configdir)/hooks
 fixesdir=$(configdir)/fixes
@@ -58,8 +58,8 @@
 	$(do_subst) < $< > $@
 
 install-exec-local:
-	@mkdir_p@ $(variantsdir)
-	touch $(variantsdir)/default.conf
+	@mkdir_p@ $(DESTDIR)$(variantsdir)
+	touch $(DESTDIR)$(variantsdir)/default.conf
 
 install-exec-hook:
 	@echo
